### PR TITLE
README.md: Add link to the wiki and the discord server

### PR DESCRIPTION
**Pull request by MaksymPrymierov**

### CODE REVIEW

**Review by rsglobal — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Manifest for GloDroid
 
-Free and opensource AOSP based Android for the world's most accessible development platforms.  
+Free and opensource AOSP based Android for the world's most accessible development platforms.
+
+GloDroid [wiki](https://github.com/GloDroid/glodroid_manifest/wiki). You could find GloDroid community discord chat [here](https://discord.gg/2z5VUv6).
```
Comment (rsglobal) [inline README.md:5]: Wiki page desn't have much useful information yet. Lets cleanup it first before publishing a link.

**Comment by e-yes** [comment]
I've spent some time finding this PR. It's nice to have all contract info in README, so please merge it;)

**Comment by rsglobal** [comment]
@MaksymPrymierov ,

Our wiki still not up-to date. Please remove wiki link and keep discord only.

**Comment by hamishmb** [comment]
Is the wiki more up to date now?

I wouldn't be opposed to updating it myself if that would enable this to be merged.

**Comment by rsglobal** [comment]
Pulled into the next branch, wiki link dropped.

**Pull request closed without merging**